compiler/noirc_evaluator/src/ssa/ssa_gen/program.rs

```diff
@@ -101,13 +107,39 @@ impl Ssa {
 
 impl Display for Ssa {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        if self.globals.dfg.values_iter().len() > 0 {
+            write!(f, "{}", self.globals)?;
+        }
+
         for function in self.functions.values() {
             writeln!(f, "{function}")?;
         }
         Ok(())
     }
 }
 
+impl Display for GlobalsContext {
```

Comment:
Can you move this Display impl (and probably Display for Ssa as well) to the ssa printer module?

Reply:
There is no longer a `GlobalsContext` wrapper struct but I have moved `Display for Ssa` to the printer module.